src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -15,13 +15,17 @@ docs_area: manage
 - The [Physical Replication dashboard](#db-console) on the DB Console.
 - [Prometheus and Alertmanager](#prometheus) to track and alert on replication metrics.
 
+When you complete a [cutover]({% link {{ page.version.version }}/cutover-replication.md %}) to the primary cluster, there will be a gap in the primary cluster's metrics whether you are monitoring via the [DB Console](#db-console) or [Prometheus](#prometheus).
```

Comment:
Is "to the primary cluster" correct? The link goes to a page with the title "Cut Over from a Primary Cluster to a Standby Cluster". So it is a little confusing.

Reply:
I deleted this part of the section as primary cluster did not make sense there. Main idea to communicate is cutover.
Thanks for catching Florence!